cmd/gateway-azure.go

```diff
@@ -590,6 +601,9 @@ func (a *azureObjects) NewMultipartUpload(bucket, object string, metadata map[st
 	if metadata == nil {
 		metadata = make(map[string]string)
```

Comment:
Neither deleting from a nil map or ranging over a nil map are errors - so it is not clear why this allocation is needed.

Reply:
Yes i know, old code not removing in this PR.